sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image-bitmap-from-blob.js

```diff
@@ -31,17 +31,36 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
         gl.clearColor(0,0,0,1);
         gl.clearDepth(1);
 
-        var xhr = new XMLHttpRequest();
-        xhr.open("GET", resourcePath + "red-green-semi-transparent.png");
-        xhr.responseType = 'blob';
-        xhr.onload = function() {
-            var blob = xhr.response;
-            runImageBitmapTest(blob, 0.5, internalFormat, pixelFormat, pixelType, gl, tiu, wtu, false)
+        fetch(resourcePath + "red-green-semi-transparent.png")
+            .then(response => {
```

Comment:
`then` is sooo 2018. :)
We should be writing this with `await`:
```
let response, blob;
debug('*** Running tests against red-green-semi-transparent.png ***');
response = await fetch(resourcePath + "red-green-semi-transparent.png");
blob = await response.blob();
await runImageBitmapTest(blob, 0.5, internalFormat, pixelFormat, pixelType, gl, tiu, wtu, false);

debug('*** Running tests against red-green-128x128-linear-profile.jpg ***');
response = await fetch(resourcePath + "red-green-128x128-linear-profile.jpg");
blob = await response.blob();
// This test requires a huge tolerance because browsers - at least currently - vary
// widely in the colorspace conversion results for this image.
const tolerance = 120;
await runImageBitmapTest(blob, 1.0, internalFormat, pixelFormat, pixelType, gl, tiu, wtu, false, tolerance);
```

Reply:
Thanks for the suggestions. :) I should have realized that possible transformation. Done throughout, and tested and removed the check of response.ok. (The tests will fail later if the image couldn't be fetched.)